new-version-bundle

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+set -eu
```

Comment:
What is this script?

Reply:
ouch :-( it was the script to automate the new version bundle. going to remove it